### PR TITLE
OSDOCS-7345: add RN entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-7345

Link to docs preview: https://73023--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes#ocp-4-16-networking-multiple-cidr

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -88,6 +88,12 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-networking"]
 === Networking
 
+[id="ocp-4-16-networking-multiple-cidr"]
+==== Define multiple CIDR blocks for one network security group rule
+
+With this release, you can now create multiple CIDR blocks with one network security group (NSG) for Ingress controllers in {product-title} clusters hosted on Azure. You can do this using the `allowedSourceRanges` field. Creating multiple Ingress controllers with the `allowedSourceRanges` field configured will allow those Ingress controllers to share the maximum limit of 4000 rules between them.
```
Comment (jldohmann) [inline release_notes/ocp-4-16-release-notes.adoc:94]: @gcs278 can you PTAL at this RN entry? would appreciate your feedback, thank you 😄

Comment (gcs278) [inline release_notes/ocp-4-16-release-notes.adoc:94]: Sure, thanks for writing this up.

Just a nit pick, I think the idea of multiple CIDR blocks in an NSG is a bit of implementation detail in this situation as the end user would probably never see this happening (unless they view the Azure NSGs directly). I think the release note should avoid directly implying that "you" can create multiple CIDR blocks in an NSG, and imply more that it's something that Azure improved and happens in the background when you use `allowedSourceRanges` on ingress controllers.

```suggestion
With this release, IP addresses and ranges are handled more efficiently in network security groups (NSG) for {product-title} clusters hosted on Azure. As a result, the maximum limit of CIDRs for all Ingress Controllers in Azure clusters, using the `allowedSourceRanges` field, has increased from approximately 1000 to 4000 CIDRs.
```

Feel free to wordsmith or adapt this to what make sense, or let me know if you disagree.

Comment (jldohmann) [reply release_notes/ocp-4-16-release-notes.adoc:94]: Excellent, this clears a lot up. Thank you!

**Comment by jldohmann** [comment]
@lihongan PTAL for QE ack. this docs PR adds a release note entry for multiple CIDRs blocks per NSG, in OCP 4.16

**Comment by lihongan** [comment]
LGTM. Thank you!

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -88,6 +88,11 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-networking"]
 === Networking
 
+[id="ocp-4-16-networking-multiple-cidr"]
+==== Define multiple CIDR blocks for one network security group (NSG) rule
+
+With this release, IP addresses and ranges are handled more efficiently in NSGs for {product-title} clusters hosted on Azure. As a result, the maximum limit of CIDRs for all Ingress controllers in Azure clusters, using the `allowedSourceRanges` field, increases from approximately 1000 to 4000 CIDRs.
```
Comment (xenolinux) [inline release_notes/ocp-4-16-release-notes.adoc:94]:
```suggestion
With this release, IP addresses and ranges are handled more efficiently in NSGs for {product-title} clusters hosted on Azure. As a result, the maximum limit of CIDRs for all Ingress Controllers in Azure clusters, using the `allowedSourceRanges` field, increases from approximately 1000 to 4000 CIDRs.
```

Comment (xenolinux) [reply release_notes/ocp-4-16-release-notes.adoc:94]: Should "Ingress controllers" be "Ingress Controllers"? Per https://redhat-documentation.github.io/supplementary-style-guide/#ingress-controller

**Review by xenolinux — COMMENTED** [review]
One question; otherwise LGTM 

A gentle reminder to add a preview link

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

File: release_notes/ocp-4-16-release-notes.adoc (hunk 2 above)
Comment (jldohmann) [reply release_notes/ocp-4-16-release-notes.adoc:94]: yes, good catch!

**Pull request merged**